pkg/command/context.go

```diff
@@ -1002,3 +1007,59 @@ func displayContextListOutputSplitViewTarget(cfg *configtypes.ClientConfig, writ
 		outputWriterTMCTarget.Render()
 	}
 }
+
+var getCtxTokenCmd = &cobra.Command{
+	Use:    "get-token CONTEXT_NAME",
+	Short:  "Get the valid CSP token for the given UCP context.",
+	Hidden: true,
+	RunE:   getToken,
+}
+
+func getToken(cmd *cobra.Command, args []string) error {
+	var name string
+	if len(args) == 0 {
+		ctx, err := promptCtx()
```

Comment:
q: is this command always meant to be run non-interactively? Is so should we support this?

Reply:
Good point. It is meant to be run non-interactively for kubectl exec plugin use case. Since the kubeconfig generated by CLI for UCP is guaranteed to have the <contextName> argument, it should not hit the issue.  However, I will remove this(and add validation for single argument) and we can add it back in future if we unhide this command in future.